app/src/main/java/org/fossasia/susi/ai/adapters/recycleradapters/ChatFeedRecyclerAdapter.java

```diff
@@ -483,18 +483,17 @@ public boolean onLongClick(View view) {
     private void handleItemEvents(final WebSearchHolder websearchholder, final int position)  {
         websearchholder.backgroundLayout.setBackgroundColor(ContextCompat.getColor(currContext, isSelected(position) ? R.color.translucent_blue : android.R.color.transparent));
 
-        webquery = MainActivity.webSearch;
-
         final ChatMessage model = getData().get(position);
+        int checkIndex = model.getContent().lastIndexOf("#");
         websearchholder.messageStar.setVisibility( (model.isImportant()) ? View.VISIBLE : View.GONE);
-        websearchholder.text.setText(model.getContent());
+        websearchholder.text.setText(model.getContent().substring(0,checkIndex));
```

Comment:
We are showing websearch when we are not getting reply from susi and that case text is simply I don't know how to answer this here is the websearch result: . There is no scope for #. Can you provide any ?